modules/internal/Bytes.chpl

```diff
@@ -426,9 +495,10 @@ module Bytes {
     inline proc param numBytes param do
       return __primitive("string_length_bytes", this);
 
-    inline proc param c_str() param : c_string {
-      return this:c_string; // folded out in resolution
-    }
+    // TODO: Support param c_ptr/c_ptrConst
```

Comment:
FWIW, I don't think it's possible for us to support this ever, so probably best to just remove this commented-out code.